native/src/main/java/io/ballerina/stdlib/mqtt/caller/CallerActions.java

```diff
@@ -41,4 +45,23 @@ public static Object complete(BObject callerObject) {
         return null;
     }
 
+    public static Object respond(BObject callerObject, BMap message) {
```

Comment:
Since this is a network action don't we hv to execute it in a non-blocking way ?

Reply:
+1

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-mqtt/pull/9/commits/041d5b9016d02da09f39901fe4d4970544edfb25